### ISSUE TITLE
Option to ignore pinned tweet while scraping user profile

### ISSUE THREAD

**Issue by PeschykhinLev** [JustAnotherArchivist/snscrape#784]

**Comment by JustAnotherArchivist** [comment]
It's not currently configurable.

**Comment by k0lter** [comment]
If an old pinned tweet exists, it breaks the `--since` argument.

Imagine an old tweet with 2023-04-01 as date. If you want to get the tweets as of today (`snscrape --since "2023-04-21" twitter-profile <user>`), snsscrape will return no results.

The pinned tweet is the first tweet returned and as the date is before the `--since` argument, snsscrape will not process the other tweets.

**Comment by JustAnotherArchivist** [comment]
Yep. I think I intended at some point to insert the pinned tweet into the timeline chronologically. But I'm not sure why that never happened, and also it should still be configurable because people might want to get the exact order shown on the web page. `--since` will always be broken under that condition though as it assumes reverse chronological order.

**Comment by JustAnotherArchivist** [comment]
This is not implemented yet, but with the addition of #868, it is now possible to filter out the pinned tweets externally if desired. The returned order is still the one on the profile page, i.e. pinned tweet followed by everything else in reverse chronological order.
